packages/mobile/package.json

```diff
@@ -111,9 +111,9 @@
     "react-native-in-app-review": "4.2.1",
     "react-native-job-queue": "0.5.0",
     "react-native-keyboard-aware-scroll-view": "^0.9.5",
-    "react-native-linear-gradient": "github:react-native-linear-gradient/react-native-linear-gradient.git#9aece37cdf7d33ed52a1747b36e9e84f3c7a49ef",
+    "react-native-linear-gradient": "2.6.2",
```

Comment:
I think we had to use a specific commit for `react-native-linear-gradient` because there was an issue with the text mask gradient (like in the screen headers) where the gradient would show as a rectangle before the text mask was applied. Let's just double check that the headers look right on android!

Reply:
gotcha, yeah i checked the release, and it seems to have marcus's commit!